### PR TITLE
JI:768804 P2022-1582 Update ES standard

### PR DESCRIPTION
**Pull request by pawelwojdas**

### CODE REVIEW

**Review by Adys23 — APPROVED** [review]
LGTM

**Comment by krysia1** [comment]
LGTM 😁

**Review by krysia1 — APPROVED** [review]

**Review by Reden4005 — COMMENTED** [review]
LGTM

**Review by Reden4005 — APPROVED** [review]

File: tsconfig.base.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "importHelpers": true,
     "target": "es2015",
     "module": "esnext",
-    "lib": ["es2017", "dom"],
+    "lib": ["es2019", "dom", "DOM.Iterable"],
```
Comment (kacperChlon) [inline tsconfig.base.json:13]: Do we want/need DOM.Iterable library?

Comment (pawelwojdas) [reply tsconfig.base.json:13]: I thought that, because I needed some options, but finally I'll omit that (I found a better solutions)

**Review by Adys23 — APPROVED** [review]
LEBTMN
(looks even better to me now) 😄

**Review by piotrrawski — APPROVED** [review]

**Pull request merged**